OpenGL/OpenGL/src/Camera.cpp

```diff
@@ -34,13 +36,41 @@ void Camera::setPosition(const glm::vec3 &newPos) {
   position = newPos;
 }
 
+void Camera::setRotation(const glm::vec3& rot) {
+     glm::vec4 newCenter(0.0f,0.0f,0.0f,1.0f);
+     newCenter = glm::translate(glm::mat4(1.0f), { 0.0f,0.0f,100.0f }) * newCenter;
+     newCenter = glm::rotate(glm::mat4(1.0f), glm::radians(-rot.x), { 1.0f,0.0f,0.0f }) * newCenter;
+     newCenter = glm::rotate(glm::mat4(1.0f), glm::radians(rot.y), { 0.0f,1.0f,0.0f })* newCenter;
+
+     center = position + glm::vec3(newCenter);
+     computeView();
+     computeProjection();
+
+}
+
 void Camera::setLookAt(const glm::vec3 &position, const glm::vec3 &center) {
   this->position = position;
   this->center = center;
   computeView();
   computeProjection();
 }
 
+void Camera::goToHead(Hitbox& hitbox) {
+
+    glm::vec3 cameraRot = hitbox.character.bodyRotation + hitbox.character.getHeadProperties().localRotation;
+    setRotation(cameraRot);
+    glm::vec3 cameraPos = hitbox.pos + hitbox.character.getHeadProperties().localPosition;
+    if (hitbox.view == View::THIRD_PERSON) {
+        glm::vec4 newPos(0.0f, 0.0f, -8.0f, 1.0f);
+        newPos = glm::rotate(glm::mat4(1.0f), glm::radians(-cameraRot.x), { 1.0f,0.0f,0.0f }) * newPos;
+        newPos = glm::rotate(glm::mat4(1.0f), glm::radians(cameraRot.y), { 0.0f,1.0f,0.0f }) * newPos;
+        cameraPos += glm::vec3(newPos);
+
+    }
+    setPosition(cameraPos);
+
+}
+
```

Comment:
So I would prefer this method in character. 
the only differences should be giving `Camera&` camera in args, changing `setRotation` / setPosition` to `camera.setRotation / camera.setPosition` and similarly removing `character.`.